virtualization-formulas/guest/metadata/form.yml

```diff
@@ -0,0 +1,20 @@
+tuning:
+    $type: group
+    $help: Virtual machine performance tuning settings
```

Comment:
If `Tuning` string is to be translatable as well, it will need `$name` field as well. See https://github.com/SUSE/spacewalk/issues/15549 2nd to do.

Reply:
OK, I'll add the `$name` for it and his other friends

Reply:
I translated the other names too...